lib/core/src/lib/styles/_reference-variables.scss

```diff
@@ -27,3 +27,9 @@ $adf-ref-height: 40px;
 $adf-ref-width: 40px;
 $adf-ref-line-height: 40px;
 $adf-ref-margin-right: 8px;
+$adf-ref-metadata-property-panel-border-color: rgba(0, 0, 0, 0.12);
+$adf-ref-metadata-buttons-background-color: rgba(33, 33, 33, 0.05);
+$adf-ref-metadata-action-button-clear-color: #212328b2;
+$adf-ref-metadata-property-panel-text-color: rgba(33, 35, 40, 0.7);
+$adf-ref-metadata-property-panel-label-color: rgba(33, 33, 33, 0.24);
```

Comment:
If we remove this https://github.com/Alfresco/alfresco-ng2-components/pull/8995/files#r1375989746 then we can remove this one too

Reply:
Used in new changes, so resolving it.